v15-upgrade-log.md

```diff
@@ -29,3 +29,6 @@ Now we will upgrade to SpatialOS 15.0.0.
    result type.
 1. Remove the use of EntityAcl from code, as it's been removed. We will deal with
    fixing the snapshot and getting authority to work later.
+1. Address changes made to command request op metadata. As we have deleted the
```

Comment:
> As we have deleted the
   code for dynamically giving authority to a client, we only need to change
   some logging messages for now.

I'm deliberately not trying very hard to understand it (pretending to be dumb user) but I don't quite follow exactly how this change relates to the prior authority changes.